### PR TITLE
Updating Torq for compatibility with finspace

### PR DESCRIPTION
**Pull request by David2464**
I have added a new file named endofperiod.q in code/rdb.

This file contains the standard endofperiod function in TorQ. This change has only been made so that the endofperiod function can be overridden when finspace is integrated with it.

### CODE REVIEW

**Review by eugenetemlock — APPROVED** [review]

**Review by jonathonmcmurray — APPROVED** [review]
LGTM

File: code/rdb/rdbstandard.q [hunk 1]
```diff
@@ -11,3 +11,8 @@ moveandclear:{[fromNS;toNS;tab]
  if[tab in key fromNS;
   set[` sv (toNS;tab);0#fromNS tab];
   eval(!;enlist fromNS;();0b;enlist enlist tab)]}
+
+\d .
+
+/-set the upd function in the top level namespace
+upd:.rdb.upd
```
Comment (jonathonmcmurray) [inline code/rdb/rdbstandard.q:18]: have you made sure this works? I think `.rdb.upd` is defined in `processes/rdb.q`, which will be loaded _after_ this file, so this may fail. I think `.rdb.upd` will also usually be defined in config somewhere, but the default definition is in `processes/rdb.q` so if a user _doesn't_ have a definition in config, I think this will fail - we should probably error trap this & put a default here to be safe (same as, or maybe instead of, the one in `processes/rdb.q`)

File: code/wdb/origstartup.q [hunk 2]
```diff
@@ -8,7 +8,7 @@ startup:{[]
         .lg.o[`init;"partition has been set to [savedir]/[", (string partitiontype),"]/[tablename]/", $[writedownmode~`partbyattr;"[parted column(s)]/";""]];
         if[saveenabled;
                 //check if tickerplant is available and if not exit with error
-                        if[not .finspace.enabled;                                                                                                                      /-TODO Remove when tickerplant fixed in finspace
+		if[not .finspace.enabled;                                                       /-TODO Remove when tickerplant fixed in finspace
```
Comment (jonathonmcmurray) [inline code/wdb/origstartup.q:11]: whitespace change in this line (I think it was spaces before, and is now tabs)

File: code/processes/wdb.q [hunk 3]
```diff
@@ -527,6 +527,6 @@ upd:.wdb.replayupd;
 / - clear any wdb data in the current partition
 .wdb.clearwdbdata[];
 /- initialise the wdb process
-.wdb.startup[];
+.wdb.startup[`];
```
Comment (jonathonmcmurray) [inline code/processes/wdb.q:530]: is this change necessary? I don't think this argument will be used, so don't think this should be needed?

**Review by jonathonmcmurray — APPROVED** [review]
couple of very minor things, but LGTM

File: code/processes/wdb.q (hunk 3 above)
Comment (zanbuchanan) [reply code/processes/wdb.q]: Ah, that was my fault, I didn't realise it would run without it so I suggested it while one a call with David (just did a mini test and I now understand :) )
